metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-rsync",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Don't increase the version in the patch for the new feature. All the release bumps are automated.